src/App.js

```diff
@@ -1,12 +1,57 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import PokemonCard from "./pokemonCard/PokemonCard";
+import computerSaysNo from "./assets/computers-says-no.gif"
+import useFetch from "./customHooks/useFetch";
+
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [data, setData] = useState({});
```

Comment:
Als je de default waarde hier aanpast naar `undefined`, dan kun je makkelijker herkennen (en checken) of data al gevuld is of niet
```suggestion
    const [data, setData] = useState(undefined);
```